ios/FluentUI/Card/CardView.swift

```diff
@@ -341,54 +439,36 @@ open class CardView: UIView {
             layoutConstraints.removeAll()
         }
 
-        /// The possible text conigurations are:
+        /// The possible text configurations are:
         /// 1) Single-line Title
         /// 2) Two-line title
         /// 3) Single-line Title + single-line subtitle
         /// In all cases, the text height is: 2 * title's height
         var height: CGFloat = ceil(2 * primaryLabel.intrinsicContentSize.height)
-
-        switch style {
-        case .horizontal:
-            height += Constants.horizontalBaseHeight
-
-            layoutConstraints.append(contentsOf: [
-                iconView.centerYAnchor.constraint(equalTo: centerYAnchor),
-                primaryLabel.leadingAnchor.constraint(equalTo: iconView.trailingAnchor, constant: Constants.horizontalContentSpacing)
-            ])
-
-            // Center the title vertically if there is no subtitle
-            if secondaryText == nil && !twoLineTitle {
-                layoutConstraints.append(primaryLabel.centerYAnchor.constraint(equalTo: centerYAnchor))
-            } else {
-                layoutConstraints.append(primaryLabel.topAnchor.constraint(equalTo: topAnchor, constant: Constants.horizontalPadding))
-            }
-        case .vertical:
-            height += Constants.verticalBaseHeight
-
-            layoutConstraints.append(contentsOf: [
-                iconView.topAnchor.constraint(equalTo: topAnchor, constant: Constants.verticalPaddingTop),
-                primaryLabel.topAnchor.constraint(equalTo: iconView.bottomAnchor, constant: Constants.verticalContentSpacing),
-                primaryLabel.leadingAnchor.constraint(equalTo: iconView.leadingAnchor)
-            ])
-        }
-
+        height += size.baseHeight
         heightConstraint.constant = height
 
         layoutConstraints.append(contentsOf: [
-            widthAnchor.constraint(equalToConstant: style.width),
+            iconView.centerYAnchor.constraint(equalTo: centerYAnchor),
+            primaryLabel.leadingAnchor.constraint(equalTo: iconView.trailingAnchor, constant: Constants.horizontalContentSpacing),
+            widthAnchor.constraint(equalToConstant: size.width),
             heightConstraint,
-            iconView.leadingAnchor.constraint(equalTo: leadingAnchor, constant: Constants.paddingLeading),
             iconView.widthAnchor.constraint(equalToConstant: Constants.iconWidth),
             iconView.heightAnchor.constraint(equalToConstant: Constants.iconHeight),
-            primaryLabel.trailingAnchor.constraint(equalTo: trailingAnchor, constant: -Constants.paddingTrailing)
+            iconView.leadingAnchor.constraint(equalTo: leadingAnchor, constant: size.leadingPadding),
+            primaryLabel.trailingAnchor.constraint(equalTo: trailingAnchor, constant: -size.trailingPadding),
+            primaryLabel.topAnchor.constraint(equalTo: topAnchor, constant: size.verticalPadding)
         ])
-
-        if secondaryText != nil {
+        
```

Comment:
Validation check failed here:
/Users/runner/work/fluentui-apple/fluentui-apple/ios/FluentUI/Card/CardView.swift:462:1: error: Trailing Whitespace Violation: Lines should not have trailing whitespace. (trailing_whitespace)

Reply:
hmm I though I had swiftlint, I guess not! Ill get this all formatted.